scopes/ui-foundation/use-box/menu/index.ts

```diff
@@ -1,2 +1,11 @@
 export { Menu } from './menu';
-export type { MenuProps } from './menu';
+export type { MenuProps, ConsumeMethod } from './menu';
+
+export { Import } from './import';
+export type { ImportProps } from './import';
+
+export { Install } from './install';
+export type { InstallProps } from './install';
+
+export { Registry } from './registry';
+export type { RegistryProps } from './registry';
```

Comment:
I guess this works? It's like consume methods templates, what happens if a developer wants a different template?

Reply:
he can use the underlying component. this is more concrete